sky/clouds/oci.py

```diff
@@ -468,7 +468,9 @@ def get_credential_file_mounts(self) -> Dict[str, str]:
             api_key_file = oci_cfg[
                 'key_file'] if 'key_file' in oci_cfg else 'BadConf'
             sky_cfg_file = oci_utils.oci_config.get_sky_user_config_file()
-        except (ImportError, oci_adaptor.oci.exceptions.ConfigFileNotFound):
+        except ImportError:
+            return {}
+        except oci_adaptor.oci.exceptions.ConfigFileNotFound:
```

Comment:
Add a comment for future reference to avoid this line being collapsed into the line above in the future. : )
```suggestion
        # This exception is explicitly separated from the line above, as this function
        # can be called by `check.get_credential_file_mounts` even without oci
        # dependencies installed. Having this exception in the same line as
        # ImportError above can cause another error being raised, due to
        # oci_adaptor.oci.exceptions.ConfigFileNotFound trying to import the oci
        # dependencies again.
        except oci_adaptor.oci.exceptions.ConfigFileNotFound:
```